conf/default.yml

```diff
@@ -3,17 +3,23 @@ api_key_blue: BLUEADMIN123
 api_key_red: ADMIN123
 app.contact.dns.domain: mycaldera.caldera
 app.contact.dns.socket: 0.0.0.0:8853
+app.contact.ftp.host: 0.0.0.0
+app.contact.ftp.port: 2222
+app.contact.ftp.pword: caldera
+app.contact.ftp.server.dir: /ftp_dir
```

Comment:
can we remove the forward slash? It might be confusing for folks who think it's an absolute file path